Resources/Prototypes/_Impstation/Objectives/thief.yml

```diff
@@ -0,0 +1,453 @@
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: ChefDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: Chef
+  - type: StealCondition
+    stealGroup: ChefDrobe
+  - type: Objective
+    difficulty: 0.4
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: DetDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: Detective
+  - type: StealCondition
+    stealGroup: DetDrobe
+  - type: Objective
+    difficulty: 1
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: JaniDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: Janitor
+  - type: StealCondition
+    stealGroup: JaniDrobe
+  - type: Objective
+    difficulty: 0.5
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: SciDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: Scientist
+  - type: StealCondition
+    stealGroup: SciDrobe
+  - type: Objective
+    difficulty: 0.6
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: RoboDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: Scientist
+  - type: StealCondition
+    stealGroup: RoboDrobe
+  - type: Objective
+    difficulty: 0.6
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: GeneDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: MedicalDoctor
+  - type: StealCondition
+    stealGroup: GeneDrobe
+  - type: Objective
+    difficulty: 0.6
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: ViroDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: MedicalDoctor
+  - type: StealCondition
+    stealGroup: ViroDrobe
+  - type: Objective
+    difficulty: 0.6
+
+- type: entity
+  parent: BaseThiefStealObjective
+  id: ServiceTechFabStealObjective
+  components:
+  - type: StealCondition
+    stealGroup: ServiceTechFab
+  - type: Objective
+    difficulty: 1
+
+- type: entity
+  parent: BaseThiefStealObjective
+  id: ShiningSpringThiefStealObjective
+  components:
+  - type: NotJobRequirement
+    job: Janitor
+  - type: ObjectiveLimit
+    limit: 3
+  - type: StealCondition
+    stealGroup: ShiningSpring
+  - type: Objective
+    difficulty: 0.5
+
+- type: entity
+  parent: BaseThiefStealObjective
+  id: ShipyardComputerCircuitboardStealObjective
+  components:
+  - type: NotJobRequirement
+    job: SalvageSpecialist
+  - type: StealCondition
+    stealGroup: ShipyardComputerCircuitboard
+  - type: Objective
+    difficulty: 0.7
+
+- type: entity
+  parent: BaseThiefStealObjective
+  id: CommsComputerCircuitboardStealObjective
+  components:
+  - type: StealCondition
+    stealGroup: CommsComputerCircuitboard
+  - type: Objective
+    difficulty: 2 #woof
+
+- type: entity
+  parent: BaseThiefStealCollectionObjective
+  id: PlushieStealCollectionObjective
+  components:
+  - type: StealCondition
+    stealGroup: Plushie
+    minCollectionSize: 12
+    maxCollectionSize: 50
```

Comment:
welcome back figurine objective: that objective was removed upstream because it was more of a cargo order than an actual steal objective, and all it accomplished was outting you as a thief. this one can't even rely on verifyMapExistence because some maps have secret rooms filled with a zillion lizard plushies

Reply:
im like......... ultimately fine with this being a cargo order type of thief objective? my thought here vs the figurine objective is that we have a server culture of people who already obsessively collect and hoard plushies, and order plushies specifically for this purpose, so its playing and iterating on how players already act. plus making your thief beacon a cuddle pile is cute. the original suggestion was just for the bug plushie (thus the icon for this objective) but i opted to make it a large plushie collection objective so it isnt either an objective that shows up EXTREMELY rarely or one where you are rolling the lottery in cargo. i intentionally did not restrict cargo techs from receiving this objective also

Reply:
we'll give it a shot